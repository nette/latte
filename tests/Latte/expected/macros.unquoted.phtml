<?php
// source: %A%

class Template%a% extends Latte\Template
{

	function render()
	{
%A%?>
<span title=<?php echo Latte\Runtime\Filters::escapeHtmlAttrUnquoted($x) ?>></span>

<span title=<?php echo Latte\Runtime\Filters::escapeHtmlAttrUnquoted($x) ?> <?php echo Latte\Runtime\Filters::escapeHtmlAttrUnquoted($x) ?>></span>

<span <?php echo Latte\Runtime\Filters::escapeHtmlAttrUnquoted('title') ?>=<?php echo Latte\Runtime\Filters::escapeHtmlAttrUnquoted($x) ?>></span>
<?php
	}

}
