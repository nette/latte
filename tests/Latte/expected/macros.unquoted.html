<span title="' &amp; &quot;"></span>

<span title="' &amp; &quot;" invalid></span>

<span invalid="' &amp; &quot;" invalid="' &amp; &quot;"></span>
